models/labels/addresses/infrastructure/token_standard/labels_token_standards_fantom.sql

```diff
@@ -10,30 +10,34 @@ SELECT distinct 'fantom' AS blockchain
 , 'infrastructure' AS category
 , 'hildobby' AS contributor
 , 'query' AS source
-, date('2023-03-02') AS created_at
-, NOW() AS modified_at
+, date '2023-03-02' AS created_at
+, NOW() AS updated_at
 , 'token_standard' AS model_name
 , 'persona' as label_type
 FROM {{ source('erc20_fantom', 'evt_transfer') }} erc20
 {% if is_incremental() %}
-LEFT ANTI JOIN this t ON t.address = erc20.contract_address
-WHERE erc20.evt_block_time >= date_trunc('day', now() - interval '1 week')
+LEFT JOIN this t
+    ON t.address = erc20.contract_address
+WHERE t.address IS NULL
+    AND erc20.evt_block_time >= date_trunc('day', now() - interval '7' day)
 {% endif %}
 
 UNION ALL
 
 SELECT distinct 'fantom' AS blockchain
-, nft.contract_address AS address
+, from_hex(nft.contract_address) AS address
```

Comment:
The source model should be migrated such that this is not necessary.

Reply:
I will remove `from_hex()` and change the ref from `{{ ref('nft_fantom_transfers_legacy') }}` to `{{ ref('nft_fantom_transfers') }}`.